internal/evaluator/evaluator.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package evaluator
+
+import (
+	"context"
+
+	"github.com/open-policy-agent/conftest/output"
+)
+
+type Evaluator interface {
+	// TODO refactor not to expose Conftest type here
```

Comment:
I think this `TODO` is already addressed.

Reply:
We're returning `github.com/open-policy-agent/conftest/output.CheckResult`, that ties any calling code to conftest.